ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,199 @@
+
+### Prerequisites
+
+- A workflow is created, see [Create a workflow](01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated, see [Link](#to-be-determined).
+
+### Procedure
+
+*Workflows > Processes > Tab OVERVIEW > Select process*
+
+![Workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/Tokens.png "[Workflow]")
+
+1. Click the *Tokens* tab.  
+If the *Hide processed* toggle is active, only the tokens currently waiting to be further processed are displayed.
+
+2. Deselect the *Hide processed* toggle, if active.  
+ All tokens, including the ones that have already been processed, are displayed.
+
+3. Check the places displayed in the diagram.  
+The places are green if a token has been input (token status is processed or being processed). If no token has been input yet, the places are white.
+
+ > [Info] A place can contain a black square displaying a number. This number specifies the number of tokens waiting to be processed at that place. Besides, a number is also displayed next to the status of each action. This number refers to the number of tokens processed in this action, and it depends on the action status and the number of tokens received. For further information, see [Workflow elements](ActindoWorkFlow/UserInterface/01_WorkflowElements.md#tokens).
+
+4. Select a place in the diagram.  
+Depending on its status and the toggle settings, the token input in the selected place is displayed in the *Tokens* tab.
+
+5. Check the token status in the *Status* column.  
+The following statuses are available:
+
+  - Processed (green)
+  - Being processed (blue)
+  - Ready (grey)
+  - Error (red)
+
+
+  [comment]: <> (Oli: Andere Status? Status korrekt? Farben bei Dead, Suspended, Aborted? Vgl. Status/Farben Places/Actions/Tokens -> ähnlich aber leicht unterschiedlich. Ist es möglich zu vereinheitlichen?)
+
+  Check any relevant information about the token displayed in the following columns:  
+
+  - *Place*: The place where the token has been input.
+  - *Data*: The type of data contained in the token.
+  - *Next possible action(s)*: The next compatible action(s) where the token can be input.
+
+[comment]: <> (OLI: Next possible action/s? Data - Token, sind die beiden Synonyme, wie kann man beide im Bezug zueinander besser beschreiben?)
+
+6. Click on an action in the diagram to display further information in the *Actions* tab. For detailed information about checking action status, see [Check the process action status](#check-the-process-action-status).
+
+7. Check the *Logs* tab for further details on any process action.
```

Comment:
Schritt 6 vielleicht rausnehmen? oder Verbindung zu Tokens klarmachen

Logs vielleicht eher in Info